src/imf_reader/sdr/read_exchange_rate.py

```diff
@@ -0,0 +1,92 @@
+"""Module to read exchange rate data from the IMF's Special Drawing Rights (SDR) Valuation dataset.
+
+Read about SDR valuation at: https://www.imf.org/external/np/fin/data/rms_sdrv.aspx
+"""
+
+import requests
+import pandas as pd
+import io
+from functools import lru_cache
+from typing import Literal
+
+from imf_reader.config import logger
+
+
+BASE_URL = "https://www.imf.org/external/np/fin/data/rms_sdrv.aspx"
+
+
+def read_data():
```

Comment:
Given the different functions that get or read data, it would be good to give this a more meaningful name that hints at what data it gets (and in this case it gets rather than reads, so `get_exchange_rates_data`)